control/delaylti.py

```diff
@@ -0,0 +1,842 @@
+    but also on past inputs.
+
+    Parameters
+    ----------
+    P : PartitionedStateSpace
+        The underlying partitioned state-space representation of the system.
+    tau : array_like, optional
+        An array of time delays associated with the system.
+    **kwargs : keyword arguments
+        Additional keyword arguments for the LTI system.
+
+    Attributes
+    ----------
+    P : PartitionedStateSpace
+        The underlying partitioned state-space representation of the system.
+    tau : array_like
+        An array of time delays associated with the system.
+    A : array_like
+        The state matrix.
+    B : array_like
+        The input matrix.
+    C : array_like
+        The output matrix.
+    D : array_like
+        The direct feedthrough matrix.
+    B1 : array_like
+        The input matrix for external inputs.
+    B2 : array_like
+        The input matrix for delayed inputs.
+    C1 : array_like
+        The output matrix for external outputs.
+    C2 : array_like
+        The output matrix for delayed outputs.
+    D11 : array_like
+        The direct feedthrough matrix for external inputs to external outputs.
+    D12 : array_like
+        The direct feedthrough matrix for delayed inputs to external outputs.
+    D21 : array_like
+        The direct feedthrough matrix for external inputs to delayed outputs.
+    D22 : array_like
+        The direct feedthrough matrix for delayed inputs to delayed outputs.
+    ninputs : int
+        The number of external inputs.
+    noutputs : int
+        The number of external outputs.
+    nstates : int
+        The number of states.
+
+    Methods
+    -------
+    from_ss(sys, tau)
+        Create a DelayLTI system from a StateSpace system.
+    from_tf(sys, tau)
+        Create a DelayLTI system from a TransferFunction system.
+    size()
+        Return the number of outputs and inputs.
+    poles()
+        Compute poles of the non-delayed part of the system.
+    zeros()
+        Compute zeros of the non-delayed part of the system.
+    feedback(other=1, sign=-1)
+        Feedback interconnection between two DelayLTI systems or a DelayLTI
+        system and a static gain.
+    issiso()
+        Check if the system is single-input, single-output.
+    __call__(x, squeeze=False, warn_infinite=True)
+        Evaluate the system's frequency response at complex frequencies.
+    to_ss()
+        Convert to StateSpace (not implemented for DelayLTI).
+    to_tf()
+        Convert to TransferFunction (not implemented for DelayLTI).
+
+    Notes
+    -----
+    The way to create a DelayLTI object is by multiplying a transfert
+    function object with a delay(tau) or exp(-tau*s). For example
+
+    >>> ct.tf([1], [1,1]) * delay(1.5)
+
+    Or
+
+    >>> s = ct.tf('s')
+    >>> ct.tf([1], [1,1]) * exp(-1.5*s)
+
+    It's possible to MIMO delayed systems from arrays of SISO delayed systems,
+    see function mimo_delay
```

Comment:
It's possible to MIMO → It is possible to create

Also missing a period at the end.